pkg/services/dashboards/service/dashboard_service_integration_test.go

```diff
@@ -815,29 +817,34 @@ type permissionScenarioFunc func(t *testing.T, sc *permissionScenarioContext)
 func permissionScenario(t *testing.T, desc string, canSave bool, fn permissionScenarioFunc) {
 	t.Helper()
 
-	mock := &guardian.FakeDashboardGuardian{
+	guardianMock := &guardian.FakeDashboardGuardian{
 		CanSaveValue: canSave,
 	}
 
 	t.Run(desc, func(t *testing.T) {
 		cfg := setting.NewCfg()
-		cfg.RBACEnabled = false
 		cfg.IsFeatureToggleEnabled = featuremgmt.WithFeatures().IsEnabled
 		sqlStore := db.InitTestDB(t)
 		quotaService := quotatest.New(false, nil)
+		// TODO: is this needed for the testing of RBAC?
+		// ac := acimpl.ProvideAccessControl(sqlStore.Cfg)
```

Comment:
do i have to use the AccessControl for these tests?

Reply:
It depends on the test - if you're testing something permission related (ie, are the permissions correctly used to filter access to some resource), then you probably need the actual AC to do the permission evaluation properly. Otherwise I'd prefer the fake service, as it simplifies the setup.

Reply:
thx, wanted that question asked here to get clarification of it